### ISSUE TITLE
Exception in sv_panel_display_nodes.py

### ISSUE THREAD

**Issue by portnov** [nortikin/sverchok#2247]
## Problem statement

There is unexpected exception printed to the console while switching to the Node view.

## Steps to reproduce

1. Checkout sverchok master
2. Launch blender
3. switch to the nodes editor
4. Observe the stdout.

## Expected result

No error messages.

## Actual result

```
Traceback (most recent call last):
  File "/home/portnov/.config/blender/2.79/scripts/addons/sverchok-master/ui/sv_panel_display_nodes.py", line 370, in draw
    displayProps = context.space_data.node_tree.displayNodesProps
AttributeError: 'NoneType' object has no attribute 'displayNodesProps'

location: <unknown location>:-1
```

## Sverchok version

git master.

**Comment by portnov** [comment]
@DolphinDream , I assume this is yours? :)

**Comment by DolphinDream** [comment]
Yep.. that was due to the panel displaying even when no node tree was available :) It's fixed now on master.

**Comment by DolphinDream** [comment]
@portnov Oops.. that was due to the panel trying to display even when no node tree was available in the node editor :) It’s fixed now on master.
